Mathlib/CategoryTheory/Abelian/GrothendieckAxioms/Basic.lean

```diff
@@ -79,6 +79,48 @@ class HasExactLimitsOfShape (J : Type u') [Category.{v'} J] (C : Type u) [Catego
 attribute [instance] HasExactColimitsOfShape.preservesFiniteLimits
   HasExactLimitsOfShape.preservesFiniteColimits
 
+variable {C} in
+/--
+Pull back a `HasExactColimitsOfShape J` along a functor which preserves and reflects finite limits
+and preserves colimits of shape `J`
+-/
+lemma hasExactColimitsOfShape_transport {D : Type*} (J : Type*) [Category J] [Category D]
```

Comment:
My only question is whether `_transport` is the optimal suffix. How about
```suggestion
lemma HasExactColimitsOfShape.domain_of_functor {D : Type*} (J : Type*) [Category J] [Category D]
```
or something like that?

`transport` suggest an equivalence to me, and your statement asks for less...